server/sonar-web/src/main/js/apps/issues/components/App.js

```diff
@@ -754,22 +726,16 @@ export default class App extends React.PureComponent {
   }
 
   renderFacets() {
-    const { component, currentUser } = this.props;
+    const { component, currentUser, myIssues } = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "currentUser" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'currentUser' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AUnusedVariable)